app/src/main/java/ch/hikemate/app/MainActivity.kt

```diff
@@ -39,7 +45,21 @@ class MainActivity : ComponentActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
-    setContent { HikeMateTheme { Surface(modifier = Modifier.fillMaxSize()) { HikeMateApp() } } }
+
+    setContent {
+      HikeMateTheme {
+        val systemBarStyle by remember {
+          val defaultSystemBarColor = android.graphics.Color.TRANSPARENT
+          mutableStateOf(
+              SystemBarStyle.auto(
+                  lightScrim = defaultSystemBarColor, darkScrim = defaultSystemBarColor))
+        }
+        LaunchedEffect(systemBarStyle) {
+          enableEdgeToEdge(statusBarStyle = systemBarStyle, navigationBarStyle = systemBarStyle)
+        }
```

Comment:
Nitpick: You should mention this changes in your PR description

Reply:
You're right. Fixed!